script/tsconfig.json

```diff
@@ -3,6 +3,7 @@
     "module": "commonjs",
     "moduleResolution": "node",
     "esModuleInterop": true,
+    "resolveJsonModule": true,
```

Comment:
script/tsconfig.json

Reply:
sadly this change is needed. If you run a command like `yarn build:dev` it will use the `tsconfig.json` located in the `/scripts` folder which calls `webpack`. Without this change it cannot resolve `package.json` in `app/package-info.ts`